### ISSUE TITLE
Unable to change color for text in categories

### ISSUE THREAD

**Issue by D3m1tx** [whiplashoo/flutter_font_picker#1]
Hi,

```
Text(fontCategory,
                style: TextStyle(
                  color: isSelectedCategory ? Colors.white : null,
                )),
```
TextStyle in categories is hardcoded to white, shouldn't it be "Theme.of(context).colorScheme.onPrimary"?
There are cases where the primary color is for example yellow and the white text is not visible enough.

Regards,

**Comment by whiplashoo** [comment]
Just pushed an updated to v0.3.1 that fixes this. Thanks for the feedback!

**Comment by D3m1tx** [comment]
Thanks, that's solved the problem, but i have the same issue on line 201, could you take a look at it when you have time?
`Text(
                                  variant,
                                  style: TextStyle(
                                      fontStyle: variant == "italic"
                                          ? FontStyle.italic
                                          : FontStyle.normal,
                                      color: isSelectedVariant
                                          ? Colors.white
                                          : null),
                                ),` 
Sorry, i didn't see it until now :(

**Comment by whiplashoo** [comment]
OK, that's fixed in v0.3.2 that I just published. If you find other issues, let me know!

Eventually, I will add the ability for the user to explicitly pass and override every color of the font picker UI.

**Comment by D3m1tx** [comment]
Thank you very much for your work. The update solves all the problems I was facing.
